packages/instantsearch.js/src/components/Template/Template.tsx

```diff
@@ -110,7 +110,11 @@ See: https://www.algolia.com/doc/guides/building-search-ui/upgrade-guides/js/#up
 
     // This is to handle Hogan templates with Fragment as rootTagName
     if (RootTagName === Fragment) {
-      return <RawHtml content={content} key={content} />;
+      return content.trim().charAt(0) === '<' ? (
```

Comment:
```suggestion
      return content.trim()[0] === '<' ? (
```

the first character is ascii so this is easier